src/node/markdownToVue.ts

```diff
@@ -43,6 +43,7 @@ export function createMarkdownToVueRenderFn(
       title: inferTitle(frontmatter, content),
       frontmatter,
       headers: data.headers,
+      relativePath: file.replace(/\\/g, '/'),
```

Comment:
@sin-tanaka What does this represent? Why are you replacing `/` with `/`? is it for Windows? I saw it in the vuepress code base too